### PR TITLE
Enhance documentation with Token permissions

### LINKED ISSUE #54: Enhance documentation - Token permissions
It may be worth to document the minimum API token requirements needed for creating the new Dynatrace config using this new "as code" tool

### PR DESCRIPTION
**Pull request by CruzanCaramele**
This commit enhances the documentation by adding the required token
permissions for each API.

fixes #54

### CODE REVIEW

**Review by didiladi — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -322,33 +322,32 @@ projects/
                                          config2.json
 ```
 
-#### Supported Configuration Types
+#### Supported Configuration Types and Token Permissions
 
 Supported configurations types are:
```
Comment (UnseenWizzard) [inline README.md:327]: Could we extend this intro line to make it clear that these are the configurations and the permissions that your env-token (making sure to reuse the wording we use in the rest of the documentation - not 100% this is it) will need to have

**Comment by UnseenWizzard** [comment]
looks like something went wrong when rebasing? There's e.g. @didiladi 's commit in there now

**Comment by CruzanCaramele** [comment]
@UnseenWizzard i rebased with **main** before pushing, would that be the cause of this ?

**Comment by tatomir146** [comment]
Let's add a Link to the official Token management documentation here? 
https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/

**Review by UnseenWizzard — DISMISSED** [review]

**Comment by CruzanCaramele** [comment]
@UnseenWizzard  @didiladi  @tatomir146  conflicts resolved, please approve

**Review by didiladi — APPROVED** [review]

**Pull request merged**